pkg/finder/fsfinder.go

```diff
@@ -55,6 +57,27 @@ func WithDepth(depthVal int) FSFinderOptions {
 	}
 }
 
+// AddFiles adds additional files with specified formats to FSFinder
+func AddFiles(files []string) FSFinderOptions {
+	return func(fsf *FileSystemFinder) {
+		if len(files) > 0 {
+		fsf.AdditionalFiles= make(map[string]string)
+			for i := range files {
+				kv := strings.Split(files[i], ":")
+				if len(kv) == 2 {
```

Comment:
I think we need to let the user know if they didn't correctly format the string as `file:type` OR if the extension is invalid, i.e. trying to scan a type we don't support. I believe this should fail hard with an error to the user that the program wasn't able to parse the `--additional-files` input

Reply:
It also may be unclear that you have to provide a full or relative path for the file in `--additional-files`